operator/pkg/util/util.go

```diff
@@ -74,19 +74,20 @@ func (d *Downloader) Read(p []byte) (n int, err error) {
 	return
 }
 
+var httpClient = http.Client{
+	Timeout: 60 * time.Second,
+}
```

Comment:
Hi @mohamedawnallah why change it to a global variable?

Reply:
I changed `httpClient` to be a global variable in order to be able to mock it in the unit tests here ?
https://github.com/mohamedawnallah/karmada/blob/e5d5c7294202b17ab7024daabe23c70716adf3aa/operator/pkg/util/util_test.go#L164

Reply:
okay, I get it.